std/math.d

```diff
@@ -4789,11 +4791,12 @@ private:
         else
             assert(0, "Not yet supported");
     }
-    static void resetIeeeFlags() @nogc
+
+    static void resetIeeeFlags()
```

Comment:
Should be `pure` per the specification:
https://dlang.org/spec/function.html#pure-functions
>8. As a concession to practicality, a pure function can also:
> * read **and write** the floating point exception flags
> ...